guide/principles.md

```diff
@@ -0,0 +1,51 @@
+# Guiding principles and rationale
+
+When deciding on style guidelines, the style team tried to be guided by the
```

Comment:
"tried" or "tries" @nrc?

Reply:
Sorry, I missed this comment earlier. I think this is correct, the guide is meant to be an archival document so when it is committed the style RFC process will be in the past and so the past tense will be correct here.

Reply:
OK, thanks! That's fine, just wanted to verify. Thanks for clarifying.